_source/user-guide/alerts/configure-alerts.md

```diff
@@ -85,6 +85,28 @@ In the _Trigger if..._ section, click **+ Add a threshold** to set up to 5 thres
 
 ![Alert trigger thresholds](https://dytvr9ot2sszz.cloudfront.net/logz-docs/alerts/alerts--trigger-settings_aug2021.png)
 
+##### Set alert schedule
+
+You can use the scheduling mechanism to manage the trigger condition frequency.
+
+Scheduling defines the frequency and the time frame for the alerts. To define a schedule, tick the On Schedule toggle and use [cron expression](https://www.freeformatter.com/cron-expression-generator-quartz.html) to specify when to trigger the alert.
```

Comment:
> To define a schedule, select **On Schedule** and use a [cron expression](https://www.freeformatter.com/cron-expression-generator-quartz.html) ....

Which is true: 
* You use a cron expression to set when the trigger conditions for an alert will be checked? 
* Alert trigger conditions are checked, and you use the cron expression to define when the alerts are sent?